tsdb/engine/tsm1/tombstone.go

```diff
@@ -323,6 +338,70 @@ func (t *Tombstoner) readTombstoneV2(f *os.File, fn func(t Tombstone) error) err
 	}
 }
 
+// readTombstoneV3 reads the third version of tombstone files that are capable
+// of storing keys and the range of time for the key that points were deleted. This
+// format is a binary and compressed with gzip.
+func (t *Tombstoner) readTombstoneV3(f *os.File, fn func(t Tombstone) error) error {
+	// Skip header, already checked earlier
+	if _, err := f.Seek(headerSize, io.SeekStart); err != nil {
+		return err
+	}
+	n := int64(headerSize)
```

Comment:
Aside from keeping track of the bytes read, you don't seem to be doing anything with `n`? I can't find a read anywhere.